packages/protobuf/src/private/names.ts

```diff
@@ -50,32 +50,23 @@ export function localName(
       const pkg = desc.file.proto.package;
       const offset = pkg === undefined ? 0 : pkg.length + 1;
       const name = desc.typeName.substring(offset).replace(/\./g, "_");
-      if (reservedIdent[name]) {
-        return name + "$";
-      }
-      return name;
+      return safeObjectProperty(safeIdentifier(name));
```

Comment:
Can you add a comment here? For example:

> For services, we only care about safe identifiers, not safe object properties, 
> but we have shipped v1 with a bug that respected object properties, and we
> do not want to introduce a breaking change.

Reply:
(thank you! :smile: I was gonna add it after we figured out the other thing)